.teamcity/settings.kts

```diff
@@ -82,6 +83,12 @@ project {
 		password("E2E_SECRETS_ENCRYPTION_KEY_AUG_22_22", "credentialsJSON:e74f4821-2033-4ef2-b9f9-95c1d7c3898a", display = ParameterDisplay.HIDDEN)
 		password("CONFIG_E2E_ENCRYPTION_KEY_LEGACY", "credentialsJSON:819c139c-90a1-4803-8367-00e5aa5fdb07", display = ParameterDisplay.HIDDEN)
 		password("E2E_SECRETS_ENCRYPTION_KEY_CURRENT", "%E2E_SECRETS_ENCRYPTION_KEY_AUG_22_22%", display = ParameterDisplay.HIDDEN)
+
+		// Calypso dashboard AWS secrets for S3 bucket.
+		password("CALYPSO_E2E_DASHBOARD_AWS_S3_ACCESS_KEY_ID", "credentialsJSON:1f324549-3795-43e5-a8c2-fb81d6e7c15d", display = ParameterDisplay.HIDDEN)
+		password("CALYPSO_E2E_DASHBOARD_AWS_S3_SECRET_ACCESS_KEY", "credentialsJSON:782b4bde-b73d-4326-9970-5a79251bdf07", display = ParameterDisplay.HIDDEN)
+		password("CALYPSO_E2E_DASHBOARD_AWS_S3_ROOT", "s3://a8c-calypso-e2e-reports")
```

Comment:
**Question:** Is this a sensitive value? I figure if it's not coming off of the credentials JSON, we probably don't need to use `password`, right? And could just use `param` or `text`?

Reply:
Right, I had intended to keep it a secret value initially, but it seemed a little unnecessary. Let's switch it back to a text.